components/Menu/src/mobile/use-togglestate.tsx

```diff
@@ -1,10 +1,9 @@
 import { useCallback, useState } from 'react';
 import { v4 as uuidv4 } from 'uuid';
 
-export const useToggleState = (callback?: () => void) => {
-  const [open, setOpen] = useState(false);
+export const useToggleState = (defaultOpen = false, callback?: () => void) => {
```

Comment:
Is er een reden om deze defaultOpen te noemen terwijl hij in MobileMenu defaultExpanded heet? Ik zou 1 optie kiezen (en vind defaultExpanded logischerder)

Reply:
Aangepast!